src/mjolnir/convert_transit.cc

```diff
@@ -47,14 +47,16 @@ namespace {
 
 // Struct to hold stats information during each threads work
 struct builder_stats {
-  uint32_t no_dir_edge_count;
-  uint32_t dep_count;
-  uint32_t midnight_dep_count;
+  uint32_t no_dir_edge_count = 0;
+  uint32_t dep_count = 0;
+  uint32_t midnight_dep_count = 0;
+  uint32_t invalid_service_dates = 0;
```

Comment:
report when there's calendar.txt entries which have 0 for all days and only add calendar_dates.txt entries which are either too far in the future or are in the past, compared with the tile creation date.